src/virtualenv/config/convert.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import, unicode_literals
+
+import logging
+
+BOOLEAN_STATES = {
```

Comment:
FWIW distutils.util.strtobool has this funny logic.

Reply:
I know, it's not stable across pythons and does not map to this entirely, and given distutils is planned to be deprecated prefer it to have it here